libs/extension-components/src/components/ShowWordContainer.svelte

```diff
@@ -0,0 +1,123 @@
+<script lang="ts">
+  import { slide } from 'svelte/transition';
+  import { createEventDispatcher } from 'svelte';
+  import { timer, merge, Subject } from 'rxjs';
+  import { take, map, toArray } from 'rxjs/operators';
+  import { Button, Dropdown } from '@parrotly.io/ui';
+
+  // bind props
+  export let showWordDurationSeconds = 4;
+
+  let disableOptions = [
+    { label: 'For 1 Hour', value: 1 },
+    { label: 'For 2 Hour', value: 2 },
+    { label: 'For 4 Hour', value: 4 },
+    { label: 'For 8 Hour', value: 8 },
+  ];
+
+  //variable declarations
+  const dispatch = createEventDispatcher();
+  const TIME_PADDING = 2;
+  const durationPlusAnimationDuration = showWordDurationSeconds + TIME_PADDING;
+  const countDown$ = timer(0, 1000).pipe(
+    take(durationPlusAnimationDuration + 1),
+    map((count) => durationPlusAnimationDuration - count)
+  );
+  const closeSubject = new Subject<null>();
+  const close$ = merge(
+    countDown$.pipe(toArray()),
+    closeSubject.asObservable()
+  ).pipe(take(1));
+
+  close$.subscribe(onClose);
+
+  function onClose() {
+    dispatch('close');
+  }
+  function onCloseClicked() {
+    closeSubject.next();
+  }
+
+  function onKnowWord() {
+    dispatch('knowWord');
+    closeSubject.next();
+  }
+
+  function handleSelect(event: CustomEvent) {
+    dispatch('disable', { durationHours: event.detail.item.value });
+    closeSubject.next();
+  }
+</script>
+
+<template>
+  <div
+    class="flex flex-col fixed dark:bg-primary-800 bg-opacity-80 dark:bg-opacity-80 bg-primary-100 p-4 pb-2 left-0 top-0 shadow-lg backdrop-blur-sm z-modal max-w-popup w-full dark:text-primary-300"
+    transition:slide={{ duration: 1500 }}
```

Comment:
```suggestion
    transition:slide={{ duration: TIME_PADDING * 1000 }}
```

Set the animation duration to the `TIME_PADDING`